Source/Projections.Store.MongoDB/ProjectionsResourceTypeRepresentation.cs

```diff
@@ -16,9 +16,8 @@ public class ProjectionResourceTypeRepresentation : IRepresentAResourceType
     {
         static readonly IDictionary<Type, Type> _bindings = new Dictionary<Type, Type>
         {
-            { typeof(IProjections), typeof(Projections) },
+            { typeof(IProjectionStates), typeof(ProjectionStates) },
```

Comment:
Didn't we have an automatic DI convention for stuff like `IClass` => `Class`?

Reply:
Yes, but this is for the resource system that was initially built to allow us in having multiple implementations for resources (like sql event store or whatever). We don't really use those capabilities since we only have a MongoDB implementation.